src/storage/storage.rs

```diff
@@ -87,12 +93,106 @@ pub trait Storage: Sync + Send {
     }
 }
 
+/// Starknet hash type.
+/// Encapsulates the result of hash functions and provides conversion functions to Cairo VM
+/// and Starknet API types for convenience.
+#[derive(Debug, Clone, Hash, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct Hash([u8; 32]);
```

Comment:
`Hash` as a name leaves a lot to the imagination...

Reply:
I know :cry: Any idea for a better name? `StarkHash` is taken.

Reply:
In 4 days I failed to come up with a suggestion, so let's live with `Hash` for now :shrug: